### ISSUE TITLE
Add support of apache2.4 commands in htaccess 

### ISSUE THREAD

**Issue by s3rj1k** [magento/magento2#10810]
Currently .htaccess template uses apache2.2 syntax

for example:

```
order allow,deny
deny from all
```

this needs to be updates for apache2.4 syntax

one could use conditional case for apache2 version like this

```
<IfVersion < 2.4> 
#... for 2.2
</IfVersion>
<IfVersion >= 2.4>
#... for 2.4
</IfVersion>
```

**Comment by TomashKhamlai** [comment]
@s3rj1k, thank you for your report.
Please use the issue reporting guidelines to report an issue. Include the steps required to reproduce, the actual result, and the expected result. See the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) for more information.

**Comment by magento-engcom-team** [comment]
@s3rj1k, thank you for your report.
We've created internal ticket(s) MAGETWO-80868 to track progress on the issue.

**Comment by jonashrem** [comment]
I'm working on it #SQUASHTOBERFEST

**Comment by mikelevy300** [comment]
Are there any plans to backport this to 2.1.x?  The system requirements for Magento 2.1.x, http://devdocs.magento.com/guides/v2.1/install-gde/system-requirements-tech.html, say Apache 2.4 is supported, but without this fix, it won't work with 2.4.  Seems like either this should be backported to 2.1.x or the documentation should be updated to say Magento 2.1.x only supports Apache 2.2.

**Comment by jonashrem** [comment]
You can install mod_access_compat to use magento 2.1 with apache 2.4. Most installations come with it enabled.

https://httpd.apache.org/docs/2.4/mod/mod_access_compat.html
